inc/class_datacache.php

```diff
@@ -474,7 +474,7 @@ function update_attachtypes()
 
 		$types = array();
 
-		$query = $db->simple_select("attachtypes", "*");
+		$query = $db->simple_select('attachtypes', '*', 'enabled=1');
```

Comment:
Any reason to skip disabled `attachtypes`? I wouldn't do it, but if it's needed for something, then the `$attachtype['enabled']` check(s) are useless.